bin/run_crab_unique_users.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# With this environment, It works in lxplus7, it can requiere modifications to run elsewhere.
+source /cvmfs/sft.cern.ch/lcg/views/LCG_96python3/x86_64-centos7-gcc8-opt/setup.sh
+source /cvmfs/sft.cern.ch/lcg/etc/hadoop-confext/hadoop-swan-setconf.sh analytix
+currentDir=$(
+  cd $(dirname "$0")
+  pwd
+)
+#In lxplus, when running with acrontab, we need to set the java home
+# to a jvm with avanced encryption enabled. 
+# see https://cern.service-now.com/service-portal/view-request.do?n=RQF1380598 
+
+if [ -e "/usr/lib/jvm/java-1.8.0" ]
+then
+export JAVA_HOME="/usr/lib/jvm/java-1.8.0"
+elif ! (java -XX:+PrintFlagsFinal -version 2>/dev/null |grep -E -q 'UseAES\s*=\s*true')
+then
+    (>&2 echo "This script requires a java version with AES enabled") 
+    exit 1
+fi
+export PYTHONPATH=$PYTHONPATH:"$currentDir/../src/python"
```

Comment:
I rather prefer to avoid redefining PYTHONPATH in a script and allow to run script from any location. The PYTHONPATH should be defined outside of shell script.

Reply:
Actually we don't need it as the script doesn't depend on the CMSSpark module. I removed it.